### ISSUE TITLE
How to meassure size of object and how to add new vehicle

### ISSUE THREAD

**Issue by zaldwis** [ahmetozlu/vehicle_counting_tensorflow#22]
Hi sir its me again :)
can you teach me how to get size of object and how to add new vehicle like motorcycle and big truck.
thank you for your email respons

**Comment by zaldwis** [comment]
I think we can meassure the size of object by count width and height of rectangle box. but I dont know how to do it, can someone teach me?

**Comment by ahmetozlu** [comment]
The current model [(ssd_mobilenet_v1_coco_2017_11_17)](https://github.com/ahmetozlu/vehicle_counting_tensorflow/tree/master/ssd_mobilenet_v1_coco_2017_11_17) capable to detect motocycles, trucks or any other objects that are determined in [this list](https://github.com/ahmetozlu/vehicle_counting_tensorflow/blob/master/data/mscoco_label_map.pbtxt). Just change the source video and see the result! If you are not satisfied from the detection accuracy, you can use [another models](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md) or you can think to [fine-tune](https://www.youtube.com/watch?v=H8sXcAXrGR4) the models with your specific image data.

You can use "right", "left", "top", "bottom" variables to calculate: 

- **_bounding box (object) perimeter:_** (2*(bottom-top) + 2*(right-left))

- **_bounding box (object) area:_** ((bottom-top)*(right-left)) 

and you can reach these variables from [there ](https://github.com/ahmetozlu/vehicle_counting_tensorflow/blob/master/utils/visualization_utils.py#L171)or [there](https://github.com/ahmetozlu/vehicle_counting_tensorflow/blob/master/utils/speed_and_direction_prediction_module/speed_prediction.py#L15) but you should work on in to predict size of the objects and it would be great to get pull request from you after you develop object size prediction algorithm to improve this open source project and make its impact big!
